### PR TITLE
Sync with theupdateframework.github.io (upstream)

### PR DESCRIPTION
**Pull request by lukpueh**
Adopt latest changes from [theupdateframework/theupdateframework.github.io](https://github.com/theupdateframework/theupdateframework.github.io)

I basically did a word-diff on the md files in the contents folder in this repo and the root folder in the upstream repo, which have all the content files and manually adopted changes.

```bash
git clone git@github.com:lucperkins/theupdateframework.io.git
git clone git@github.com:theupdateframework/theupdateframework.github.io.git

for f in `ls content`; do git --no-pager diff --no-index --word-diff=color theupdateframework.github.io/$f theupdateframework.io/contents/$f | less -R; done
```

See commit messages for more details.

### CODE REVIEW

**Comment by lukpueh** [comment]
Just fixed 2 more things I stumbled across:
- Fixing in-page anchor spacing
- Update TUF's cncf project status

**Comment by lucperkins** [comment]
@lukpueh Ah, this is totally my bad. I meant to transfer everything over to the TUF org. We can do that once this gets merged. I'll review right now.

File: assets/sass/style.sass [hunk 1]
```diff
@@ -69,6 +69,14 @@ $colors: mergeColorMaps(("{{ .name }}": ({{ .hex }}, $white)), $colors)
 {{ end }}
 {{ end }}
 
+// Show h2 headings a bit below navbar when clicking anchors, without affecting
+// the spacing between a heading and a preceding paragraph.
+h2:before
```
Comment (lucperkins) [inline assets/sass/style.sass]: Nicely done

**Comment by lukpueh** [comment]
> I meant to transfer everything over to the TUF org. We can do that once this gets merged. I'll review right now.

Thanks for reviewing and merging. Transferring this repo to the theupdateframework is a good idea. Let me delete the fork.

**Pull request merged**